<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrameSelectWidget</class>
 <widget class="QWidget" name="FrameSelectWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>222</width>
    <height>159</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QSlider" name="slider3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSlider" name="slider4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QSpinBox" name="spinBox2">
     <property name="suffix">
      <string> / ZZ</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QSpinBox" name="spinBox3">
     <property name="suffix">
      <string> / TT</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QSpinBox" name="spinBox4">
     <property name="suffix">
      <string> / VV</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label2">
     <property name="text">
      <string>Dim 2 (Z)</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label3">
     <property name="text">
      <string>Dim 3 (T)</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label4">
     <property name="text">
      <string>Dim 4 (V)</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSlider" name="slider2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="4">
    <widget class="QCheckBox" name="checkCropV">
     <property name="text">
      <string>Crop last dimension (RGB/V) too</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Dim 1 and 2 (XY) stay untouched.&lt;br&gt;To swap dimensions, use the ChannelConverter.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>slider4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox4</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>311</y>
    </hint>
    <hint type="destinationlabel">
     <x>653</x>
     <y>327</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider4</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>653</x>
     <y>327</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>653</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>653</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>257</y>
    </hint>
    <hint type="destinationlabel">
     <x>653</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>653</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>FrameSelectWidget</receiver>
   <slot>setDim2(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>257</y>
    </hint>
    <hint type="destinationlabel">
     <x>45</x>
     <y>2</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>FrameSelectWidget</receiver>
   <slot>setDim3(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>41</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>FrameSelectWidget</receiver>
   <slot>setDim4(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>311</y>
    </hint>
    <hint type="destinationlabel">
     <x>35</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkCropV</sender>
   <signal>toggled(bool)</signal>
   <receiver>FrameSelectWidget</receiver>
   <slot>setCropV(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>338</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkCropV</sender>
   <signal>toggled(bool)</signal>
   <receiver>slider4</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>342</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>323</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkCropV</sender>
   <signal>toggled(bool)</signal>
   <receiver>label4</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>348</y>
    </hint>
    <hint type="destinationlabel">
     <x>45</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkCropV</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinBox4</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>667</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>675</x>
     <y>326</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setCropV(bool)</slot>
  <slot>setDim2(int)</slot>
  <slot>setDim3(int)</slot>
  <slot>setDim4(int)</slot>
 </slots>
</ui>
